backend/prisma/schema.prisma

```diff
@@ -70,6 +71,8 @@ model Plan {
   createdAt        DateTime         @default(now())
   updatedAt        DateTime         @updatedAt
   PlanMembership   PlanMembership[]
```

Comment:
This should be planMembership (my mistake) haha

Reply:
No worries! fixed